WordPress/Classes/ViewRelated/Site Creation/SiteSegments/SiteSegmentsWizardContent.swift

```diff
@@ -155,27 +157,56 @@ final class SiteSegmentsWizardContent: UIViewController {
     }
 
     private func handleError(_ error: SiteSegmentsError) {
-        setupEmptyTableProvider()
+        setupErrorView()
     }
 
-    private func setupEmptyTableProvider() {
-        let message: InlineErrorMessage
-        if isNetworkActive {
-            message = InlineErrorMessages.serverError
+    private func setupErrorView() {
+        let errorType: ErrorStateViewType
+        if self.isNetworkActive == false {
+            errorType = .networkUnreachable
         } else {
-            message = InlineErrorMessages.noConnection
+            errorType = .siteLoading
         }
+        self.installErrorStateViewController(with: errorType)
+    }
+
+    private func installErrorStateViewController(with type: ErrorStateViewType) {
+        var configuration = ErrorStateViewConfiguration.configuration(type: type)
 
-        let handler: CellSelectionHandler = { [weak self] _ in
-            self?.fetchSegments()
+        configuration.contactSupportActionHandler = nil
```

Comment:
Given that the `ErrorStateViewConfiguration` initializer includes `nil` defaults for both the `contactSupportActionHandler` & `dismissalActionHandler`, you should be able to delete lines 176 & 177 here.